### PR TITLE
Add notes on how to make it work on Windows

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

**Comment by adamedx** [comment]
:+1: , though I'm not a fan of the password stuff in general. This gives it parity with Linux builders, but it's definitely unsafe to use a box like that if there's any incoming network access to the guest.

File: README.md [hunk 1]
```diff
@@ -84,6 +84,17 @@ development. Test Kitchen also exposes the ability to provision instances using
...
 information, please see the [Test Kitchen documentation](http://kitchen.ci).
 
+For Windows instances, you need to take two extra steps:
+
+1. Modify .kitchen.yml so that the "chef" project has the correct chef_omnibus_root:
+   ```
+   - name: chef
+     provisioner:
+       chef_omnibus_install_options: -P angrychef
+       chef_omnibus_root: /opscode/angrychef
```
Comment (ksubrama) [inline README.md]: C:/opscode/angrychef

using just /opscode may select the default drive on which this command is being run which may be fine but it's better to be explicit.

File: .kitchen.yml [hunk 2]
```diff
@@ -82,6 +82,8 @@ platforms:
   - name: windows-2008r2-standard
     driver:
       box: chef/windows-server-2008r2-standard # private
+      synced_folders:
+        - ['..', '/home/vagrant']
```
Comment (ksubrama) [inline .kitchen.yml]: This is presumably to copy all the directories that are peers to the one you're in?  Windows has no /home - did you mean to land it in C:/Users/vagrant/repo or something like that?

**Comment by ksubrama** [comment]
Already merged.

**Pull request closed without merging**